### DISCUSSION TITLE
Is it possible to send new security blogs to a slack channel

### DISCUSSION THREAD

**Discussion by vyastj** [community/community#54925] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello team

Sometimes I miss reading the security blog and wanted to know if there is a way to send new updates at https://github.blog/category/security/ to a Slack channel. This would help me get notified whenever there is a new post

**Comment by airtower-luna** [comment, marked as answer]
There's an RSS feed both for the blog in general and the security category. You could subscribe to that directly, or have some tool read it and post links to new articles wherever you want.

**Reply by vyastj** [reply]
@airtower-luna Can you share the link to the RSS feed for the security category? I am not able to find it

**Reply by airtower-luna** [reply]
Sure: https://github.blog/category/security/feed/

It's one of the feeds listed in the HTML head as `<link rel="alternate" type="application/rss+xml" ...>`. I should mention I didn't need to dig into the HTML manually, I use [a Firefox add-on](https://addons.mozilla.org/de/firefox/addon/livemarks/) that automatically shows an RSS icon in the address bar if the site announces any feeds. :wink:

**Reply by vyastj** [reply]
I think there should be something similar for Chrome as well. And thanks for the help 🙇

**Reply by airtower-luna** [reply]
Glad I could help! Please don't forget to mark the actual answer. :wink:
